projects/storefrontapp/src/main.server.ts

```diff
@@ -14,4 +14,4 @@ if (environment.production) {
   enableProdMode();
 }
 export { renderModule } from '@angular/platform-server';
-export { AppServerModule as default } from './app/app.server.module';
+export { AppModuleServer as default } from './app/app.module.server';
```

Comment:
[MINOR]
Although angular renamed the path, the module name remained unchanged `AppServerModule` 🙃 

```suggestion
export { AppServerModule as default } from './app/app.module.server';
```

Reply:
You are right. Lol